protocols/kad/CHANGELOG.md

```diff
@@ -6,7 +6,10 @@
 
 - Derive `Error` for `GetRecordError` (see [PR 2614]).
 
-[pr 2614]: https://github.com/libp2p/rust-libp2p/pull/2614
+- Renamed `PeriodicJob::is_ready` to `PeriodicJob::check_ready`. See [PR 2620].
```

Comment:
This method is not exposed and thus would not need to be mentioned in the changelog, right?

Reply:
Indeed! Fixed.